File: lab-3-ddb/environment/ruby_2.6.0/create_table.rb
require "aws-sdk-dynamodb"
ddb_client = Aws::DynamoDB::Client.new(region: "<FMI>")

resp = ddb_client.create_table({
  attribute_definitions: <FMI>, 
  key_schema: [
    {
      attribute_name: "<FMI>", 
      key_type: "<FMI>", 
    }
  ], 
  provisioned_throughput: {
    read_capacity_units: <FMI>, 
    write_capacity_units: 1, 
  }, 
  table_name: "<FMI>", 
})

puts resp.to_h

File: lab-3-ddb/environment/node_8.10.0/create_table.js
var 
    AWS = require("aws-sdk"),                              
    DDB = new AWS.DynamoDB({
        apiVersion: "2012-08-10",
        region: "<FMI>"
    });                                           

(function createADataBaseTable(){
    var 
        params = {
            <FMI>: [{
                AttributeName: "petname", 
                AttributeType: "S"
            }], 
            KeySchema: [{
                AttributeName: "petname", 
                KeyType: "<FMI>"
            }], 
            ProvisionedThroughput: {
                ReadCapacityUnits: 1, 
                <FMI>: 1
            }, 
            <FMI>: "lostcats"
    };
    DDB.<FMI>(params, function(err, data){
        console.log(err, data);             
    });
})();
